examples/v1beta1/argo/README.md

```diff
@@ -0,0 +1,52 @@
+# Katib Examples with Argo Workflows Integration
+
+Here you can find examples of using Katib with [Argo Workflows](https://github.com/argoproj/argo-workflows).
+
+**Note:** You have to install `Argo >= v3.1` to use it in Katib Experiments.
```

Comment:
v3.1.3 to be consistent with below? Otherwise need to verify v3.1 works in general

Reply:
Sure, thank you for pointing this.